README.md

```diff
@@ -179,6 +184,9 @@ sudo apt install libgpgme-dev libassuan-dev libbtrfs-dev libdevmapper-dev libost
 
 # macOS:
 brew install gpgme
+
+# openSUSE
+sudo zypper install libgpgme-devel device-mapper-devel
```

Comment:
According to the [.spec file](https://build.opensuse.org/package/view_file/Virtualization:containers/skopeo/skopeo.spec?expand=1) on OBS, there are some more dependencies needed:
* glib2-devel
* libbtrfs-devel

Reply:
Yeah, I forgot these ones. I've already have these installed in my local machine. Thanks for the heads-up